libvirt/image/Makefile

```diff
@@ -120,9 +125,33 @@ $(PAUSE_SRC): $(SKOPEO_SRC)/bin/skopeo
 $(PAUSE): | $(PAUSE_SRC) $(UMOCI_SRC)/umoci
 	$(UMOCI_SRC)/umoci unpack --rootless --image "$(PAUSE_SRC):$(PAUSE_VERSION)" "${FILES_DIR}/$(PAUSE_BUNDLE)"
 
+docker-build:
+	docker build -t caa-libvirt-ubuntu-podvm-builder \
+	-f Dockerfile_ubuntu.podvm_builder .
+	docker run --rm \
+		-v $(PROJECT_DIR):/src/cloud-api-adaptor \
+		-v $(abspath $(PROJECT_DIR)/../kata-containers):/src/kata-containers \
+		-e UID=$(shell id -u) \
+		-e GID=$(shell id -g) \
+		caa-libvirt-ubuntu-podvm-builder \
+		bash -c 'trap "chown -R -f "$$UID:$$GID" output staticlib $(SKOPEO_SRC) \
+		$(UMOCI_SRC) $(PAUSE_SRC) $(FILES_DIR)/$(PAUSE_BUNDLE) $(FILES_DIR)/usr \
+		../../agent-protocol-forwarder || true" EXIT; \
+		source ~/.cargo/env && CLOUD_PROVIDER=libvirt make build'
 clean:
 	rm -f "$(IMAGE_FILE)" "$(UBUNTU_IMAGE_FILE)" $(BINARIES)
 	rm -fr "$(SKOPEO_SRC)" "$(UMOCI_SRC)" "$(PAUSE_SRC)" "$(FILES_DIR)/$(PAUSE_BUNDLE)"
+	rm -fr staticlib output
 
 .PHONY: force
 force:
+
+.PHONY: push
```

Comment:
@jtumber-ibm can you check this PR and suggest your views so that it helps with  https://github.com/confidential-containers/cloud-api-adaptor/issues/314 ?

Reply:
> @jtumber-ibm can you check this PR and suggest your views so that it helps with #314 ?

The file matches up closely to the other providers and so will be simple to migrate to the common approach. The plan for #314 is to introduce the "common" Makefile first and allow the provider SMEs to update and test each provider Makefile. As this file shares the same "common" sections I'm happy for it to go in like this before #314 is ready.

Reply:
Cool, I wasn't aware of #314 and I got just lucky because I copied the `push` idea from the ibmcloud's Makefile :)